demo-app/server/index.js

```diff
@@ -29,7 +29,7 @@ const server = http.createServer(async (req, res) => {
       nimbus_preview: nimbusPreview,
     };
 
-    let pathName = '/v1/features/'
```

Comment:
Actually just out of an abundance of caution, can we keep the test app on v1 so we ensure that v1 completely remains intact while landing this and we can bump it to v2 later? 🙏